sfge/src/main/java/com/salesforce/graph/vertex/FieldWithModifierVertex.java

```diff
@@ -33,4 +33,16 @@ public boolean isPrivate() {
     public boolean isStatic() {
         return getModifierNode().isStatic();
     }
+
+    public boolean isAbstract() {
+        return getModifierNode().isAbstract();
+    }
+
+    public boolean isVirtual() {
+        return getModifierNode().isVirtual();
+    }
+
+    public boolean isGlobal() {
+        return getModifierNode().isGlobal();
+    }
```

Comment:
Since fields can never have `virtual` or `abstract` modifier, this somehow feels a little uncomfortable. Can we extend `FieldWithModifierVertex` class to add the newer methods? Something like a "TypeDefinitionVertex"?